src/features/NewOrder/NewOrder.js

```diff
@@ -0,0 +1,300 @@
+//WORK ITEMS
+
+// Drop down suggestions for stock symbol input.
+// keep input value (watched) in state?
+// show display if match
+
+// imports
+import React, { useState, useRef } from 'react';
+import { useForm } from 'react-hook-form';
+
+// Styles
+import styles from './NewOrder.module.css';
+
+// sample data
+const realMemeStocks = ['GMSK', 'HYPE', 'MEME', 'MDPC', 'OTHR'];
+const ownedStocks = [
+  {
+    stockName: 'GMSK',
+    stockShares: 7,
+  },
+  {
+    stockName: 'HYPE',
+    stockShares: 23,
+  },
+  {
+    stockName: 'MEME',
+    stockShares: 42,
+  },
+];
+
+const totalCash = 5000.5;
+
+export default function NewOrder() {
+  // use state for showing buy or sell info
+
+  const [showBuyForm, setShowBuyForm] = useState('true');
+
+  return (
+    <div className={styles.formContainer}>
+      <h1>New Order</h1>
+      <div>
+        <button
+          className={styles.buybtn}
+          disabled={showBuyForm}
+          onClick={() => setShowBuyForm(true)}
+        >
+          Buy
+        </button>
+
+        <button
+          className={styles.sellbtn}
+          disabled={!showBuyForm}
+          onClick={() => setShowBuyForm(false)}
+        >
+          Sell
+        </button>
+      </div>
+
+      {showBuyForm ? <BuyForm /> : <SellForm />}
+    </div>
+  );
+}
+
+function BuyForm() {
+  // const [showSearchSuggestions, setSearchSuggestions] = useState("true");
+
+  const { register, handleSubmit, watch, reset, errors } = useForm({
+    mode: 'onBlur',
+    reValidateMode: 'onBlur',
+  });
+
+  const sharePrice = useRef({});
+  sharePrice.current = watch('price', '0.00');
+
+  const quantity = useRef({});
+  quantity.current = watch('quantity', '');
+
+  let totalPrice = sharePrice.current * quantity.current;
+
+  let cashRemaining = totalCash - totalPrice;
+
+  const onSubmit = (data) => {
+    console.log(data);
+    reset();
+  };
+  console.log(errors);
+
+  function findMatch(value) {
```

Comment:
You did a good job coming up with this function. You may want to look into the `find` and `includes` methods for arrays. See comment bellow in the validator.